Packs/Netskope/Integrations/NetskopeEventCollector/NetskopeEventCollector.py

```diff
@@ -36,23 +36,29 @@ def __init__(self, base_url: str, token: str, api_version: str, validate_certifi
         else:
             self.headers = {'Netskope-Api-Token': token}
 
-    def get_events_request_v1(self, event_type: str, last_run: dict, limit: Optional[int] = None) -> Dict:
+    def get_events_request_v1(self, event_type: str, last_run: dict, skip: int = None,
+                              limit: int = None, is_command: bool = False) -> Dict:
         body = {
             'starttime': last_run.get(event_type),
             'endtime': int(datetime.now().timestamp()),
-            'limit': limit,
-            'type': event_type
+            'limit': limit if is_command else MAX_EVENTS_PAGE_SIZE,
```

Comment:
Yes. Basically I thought it will be best to limit the results via the API when calling the regular commands/test_module.
I can remove it, and always get the MAX_PAGE_SIZE and then limit what returns to the user afterward (as we're doing when sending the events).
Whatever you think will work best

Reply:
Not sure I fully understood, let's quickly sync on it tomorrow.